functests/1_performance/performance.go

```diff
@@ -1079,6 +1079,18 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 			})
 		})
 	})
+
+	Context("Verify containers specifications", func() {
+
+		// Automates https://bugzilla.redhat.com/show_bug.cgi?id=2055019
```

Comment:
I think this is not neccesary

Reply:
Removed in the new PR target 4.10 branch.
https://github.com/openshift-kni/performance-addon-operators/pull/934

Thank you.